x11/font-spleen/Portfile

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
```

Comment:
Fix lint warning:
```suggestion

PortSystem          1.0
```

Reply:
Thanks, will force push in a minute. How do you get the lint warning for the above? With the current HEAD,
```shell
❯ git diff && port lint
--->  Verifying Portfile for font-spleen
--->  0 errors and 0 warnings found.
```

Reply:
the CI does a bit stricter checking using the command `port lint --nitpick`